sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -16589,7 +16589,7 @@
               "inputFormat": "plain",
               "validate": {
                 "required": true,
-                "custom": "",
+                "custom": "valid = (input >= 0 && input < 100000000) ? true : 'The number you have entered is not an acceptable value.'",
```

Comment:
Was this message confirmed with the biz?

Reply:
@andrepestana-aot during grooming we threw a number of what max value could be and decided 100M

Reply:
Yes @andrepestana-aot , the message (added by biz) is there in the story now. Thanks!